mesonbuild/dependencies/base.py

```diff
@@ -520,8 +520,12 @@ def __repr__(self):
                         self.version_reqs)
 
     def _call_pkgbin_real(self, args, env):
-        p, out = Popen_safe(self.pkgbin.get_command() + args, env=env)[0:2]
-        return p.returncode, out.strip()
+        cmd = self.pkgbin.get_command() + args
+        p, out = Popen_safe(cmd, env=env)[0:2]
+        rc, out = p.returncode, out.strip()
+        call = ' '.join(cmd)
+        mlog.debug("Called `{}` -> {}\n{}".format(call, rc, out))
```

Comment:
You probably want to print this before calling the command so that it's printed even when the call fails in a way that raises an unhandled exception.